common/infra-virt.function

```diff
@@ -48,7 +48,7 @@ upload_logs() {
     source ~/openrc
     BUILD_PLATFORM=${BUILD_PLATFORM:-"unknown_platform"}
     CONTAINER=${CONTAINER:-"unknown_platform"}
-    for path in /var/lib/edeploy/logs /var/log  /var/lib/jenkins/jobs/puppet/workspace; do
+    for path in /var/lib/edeploy/logs /var/log  /var/lib/jenkins/jobs/puppet/workspace /var/lib/jenkins/jobs/builds/1; do
```

Comment:
Good point.

For Puppet job the logs are not printed by jenkins console they are instead wrote in the workspace directory of the job by the script configure.sh.

This is not the case for sanity job because the actual tempest logs are printed on jenkins console so on builds/1.